app/models/school_local_authority_district.rb

```diff
@@ -1,5 +1,27 @@
 # frozen_string_literal: true
 
+# == Schema Information
+#
+# Table name: school_local_authority_districts
+#
+#  id                          :uuid             not null, primary key
+#  end_year                    :integer
+#  start_year                  :integer          not null
+#  created_at                  :datetime         not null
+#  updated_at                  :datetime         not null
+#  local_authority_district_id :uuid             not null
+#  school_id                   :uuid             not null
+#
+# Indexes
+#
+#  index_school_local_authority_districts_on_school_id  (school_id)
+#  index_schools_lads_on_lad_id                         (local_authority_district_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (local_authority_district_id => local_authority_districts.id)
+#  fk_rails_...  (school_id => schools.id)
+#
```

Comment:
If we can see the id names in the comment above, and there is a relation on the model saying what this model `belongs_to`, do we gain anything by this foreign key comment?